### ISSUE TITLE
Trouble with the wifi server

### ISSUE THREAD

**Issue by mrkurger** [joelsernamoreno/EvilCrowRF-V2#49]
Hi, I|ve gotten through your instructions all the way to what I thought would be the simplest part, putting the files on the SD card and logging on to the web server.

But it seems this is where I'll be banging my head for a little while.

Running Kali, and using a fairly new 32gb microSD card which works in every other usecase. Have tried formatting the SD card as both FAT and ext4, but I just cant seem to get any response from 192.168.4.1

Oh, and by the way, is a Discord invite possible? The one on the repo front page has expired.

**Comment by joelsernamoreno** [comment]
Hello!

Some 32 GB microSD cards do not work properly, I use a 32 GB SanDisk and it works fine

Can you try a smaller size card? 16 GB, 8 GB?

Discord: https://discord.gg/5dYZv9rS

**Comment by Hackstur** [comment]
Hi! i think i have the same problem... i sucesfully flash the device with the firmware.. and i can see the bssid "Evil Crow RF v2", connect, but when i try to opem http://192.168.4.1 always get a 404 error. Mi card size is 16GB. Any clue?

bytheway: the discord link are expired again, can you send new one? :D

**Comment by joelsernamoreno** [comment]
Hello!

Yes, some microSD cards do not work properly

Discord link: https://discord.gg/CV5hD4wx

**Comment by majestic23** [comment]
Hi can anyone update the discord link?

**Comment by joelsernamoreno** [comment]
Hi @majestic23 

https://discord.gg/rHWrV6m9

**Comment by kaleebugan** [comment]
has anyone update the discord link? i got the same problem. i flashed it correctly. i followed the step repeteadly but the SSID cannot be search on my phone. i hope someone can help :(

**Comment by joelsernamoreno** [comment]
Hello @kaleebugan 

https://discord.gg/CWpQ5wVU

If you have flashed the firmware and do not see the wifi access point, try changing the WiFi channel in the firmware.

Open firmware.ino and look for this:

const int wifi_channel = 

Change the WiFi channel number to another WiFi channel. Some mobile phones do not work on some WiFi channels

Change this and re-flash the firmware

**Comment by kaleebugan** [comment]
ok thank you very much sir will do that now. i will give an update asap

**Comment by kaleebugan** [comment]
where exactly i will put the desire ssid name? this is the code below.

// Config SSID, password and channel
const char* ssid = "Evil Crow RF v2";  // Enter your SSID here
const char* password = "123456789ECRFv2";  //Enter your Password here
const int wifi_channel = 1; //Enter your preferred Wi-Fi Channel

**Comment by joelsernamoreno** [comment]
Do you mean for AP mode or STA mode? Do you want ECRF V2 to have a wifi access point or ECRF V2 to connect to a wifi access point?

If you change this, ECRF V2 will use another SSID and password for AP. If you want ECRF V2 to connect to your home wifi access point (for example), this is easiest to do from the web panel

**Comment by kaleebugan** [comment]
ok i geit it now sir. thank you i will change the SSID and password now and give an update asap

**Comment by joelsernamoreno** [comment]
Yes, I am programming the firmware from scratch because this is broken due to the latest library updates

You can find a compiled firmware inside v1.3.2. You can flash this with some tool like ESP-Flasher

You can also use other alternative firmwares:

https://github.com/h-RAT/EvilCrowRF_Custom_Firmware_CC1101_FlipperZero

https://github.com/tutejshy-bit/tut-rf

**Comment by kaleebugan** [comment]
i dont know what the problem is :( im hopeless now.
the device is on but the AP is not on network list to connect on it sir :(

![Image](https://github.com/user-attachments/assets/70778a66-0519-4f8e-9c5a-1a5bf6bff9f5)

![Image](https://github.com/user-attachments/assets/4609dbda-d3e0-41c3-883e-038af4d5c63c)

**Comment by joelsernamoreno** [comment]
Read my previous comment

Anyway, you do not have the original device. You have a clone of Evil Crow RF V2

**Comment by joelsernamoreno** [comment]
I do not know what is inside that device because it is not the original device

**Comment by joelsernamoreno** [comment]
Maybe you have not been scammed, there are other people who have bought this clone and it has worked, but it has not worked for many other people

Try flashing the .bin firmware from v1.3.2 or the h-rat firmware and pray that this works

**Comment by kaleebugan** [comment]
will try that sir thank you so much for your help and time. i really appreciated it

**Comment by docrod** [comment]
Hi! I am also having the same problem that Hackstur posted above:

_"i think i have the same problem... i successfully flash the device with the firmware.. and i can see the ssid "Evil Crow RF v2", connect without problems with the right password..., but when i try to open http://192.168.4.1/ always get a 404 error. Mi card size is 16GB. Any clue?

bytheway: the discord link are expired again, can you send new one? :D

EDIT: My fault... the problem is the SD itself, changed it... and now it works fine!" Hackstur_

For my experience, everything flashed, including your most recent upload (1.3.3), with a tweak of the libraries and an export of the .bin file via arduino, followed by an upload of the .bin file via elegantOTA webserver on the ECRFv2. I copied over the corresponding files in the SD directory to the SD card, but I get a connection error when I try to access the IP: 192.168.4.1 after connecting to the AP. 

Would you be able to send a new discord link? I am especially curious to see if someone has compiled a list of compatible SD cards. I am currently using a 32Gb SanDisk without any luck. Any ideas?

Thanks!

**Comment by joelsernamoreno** [comment]
Hello @docrod 

If you want you can send PR and update firmware v1.3.3 with your change

Yes, discord link:

https://discord.gg/KhNQ3Cwf

**Comment by docrod** [comment]
[EvilCrow_v1.3.3.1.zip](https://github.com/user-attachments/files/20218476/EvilCrow_v1.3.3.1.zip)

Hi, Joel. 

Please see attached. 

Thanks!

**Comment by joelsernamoreno** [comment]
Thanks @docrod 

I will update this next week. I will include you in the contributors section

**Comment by docrod** [comment]
Great. Thanks!

**Comment by joelsernamoreno** [comment]
Hello

I’ve released a new firmware update.
I’m going to close all existing issues. If you encounter any problems with the new firmware, or if any previous issues haven’t been resolved, please open a new issue.

Thank you!

**Comment by Variable-61** [comment]
Hi Joel,

Thank you for the great project.
I have an issue, purchase ECRFV2 from Amazon, it came already set up porabbly with a very early sw version which managed to follow instructions and using VSC flashed and updated it to the Flipper Zero version and it is working great. Problem is I purchased another from another seller with no SDcard or sw installed. I followed your instructions and thought did everything using Arduino ino, flashed and copied folder to SD but it is not running. I am thinking it might be missing a boot loader maybe - with the SD card in or out I just get the LED next to the USB light up reletively dimmed pretty quickly after power up - I dont get the webserver running, Just the green LED is on pretty quickly aftre power up whether the sd card is in or not, so I thought maybe the bootloader is missing, but I have no experience with bootloader.

Any help wuld be very much appreciated.
Thanks very much.
Sean

**Comment by Variable-61** [comment]
An update. I used ESPhome flasher to upload an old firmware v1.3.2, once that went in it started to work with the old firmware ver 1.0 but I would like to put the latest FW. When I try to compile the latest FW.ino I just get errors such as:

153 |     #error configNUMBER_OF_CORES must be defined to either 1 or > 1.
959 |     #error configTASK_NOTIFICATION_ARRAY_ENTRIES must be at least 1
74 | #error BOOT_PIN not defined for this chip!
15 | #error "This header should only be included when building for ESP32"
649 | #error "One of CONFIG_TCP_OVERSIZE_xxx options should be set by sdkconfig
1721 | #error "Please enable at least one IP stack (either IPv4 or IPv6 or both)"

So nocompile.

Thank you for your help.
